.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1 +1,5 @@
 blank_issues_enabled: false
+contact_links:
+  - name: Discord
+    url: https://discord.gg/GyCBjQhqCd
+    about: "Ask for support in our Discord Server!."
```

Comment:
I rather not forward users to Discord, which turns into a mess for anything that might might eventually be turned into an issue.

We have discussions enabled in this repository, that's probably a better place to redirect these types of questions.